test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -142,16 +142,28 @@ bootstrap_remote_freebsd()
 
     if [ "${controller}" ]; then
         jinja2_pkg="py${python_package_version}-jinja2"
-        cryptography_pkg="py${python_package_version}-cryptography"
         pyyaml_pkg="py${python_package_version}-yaml"
+        cryptography_pkg="py${python_package_version}-cryptography"
+		packaging_pkg="py${python_package_version}-packaging"
```

Comment:
Why is this needed?

Reply:
Interesting. I see it's available on FreeBSD 13.2 as well (for Python 3.9, not 3.11). This is good to have, it just needs indention updated to match the other variables. I wonder if we missed this when adding FreeBSD 13.2, or if it was added later.